react/features/welcome/components/WelcomePage.native.js

```diff
@@ -54,7 +80,13 @@ class WelcomePage extends AbstractWelcomePage {
     componentWillMount() {
         super.componentWillMount();
 
-        this.props.dispatch(createDesiredLocalTracks(MEDIA_TYPE.VIDEO));
+        const { dispatch } = this.props;
+
+        if (this.props._profile.startAudioOnly) {
+            dispatch(destroyLocalTracks());
```

Comment:
So on native by this point the audio track has always been destroyed? Just wondering.

Reply:
Not sure about the audio, I haven't checked that. The purpose if this is to basically destroy every tracks, so we start with a clear state. Good question though, I'll check